docs/rst/tutorials/cloud/wan_edge_cloud/wan_edge_cloud.rst

```diff
@@ -336,4 +336,10 @@ and the ``turtlesim_square_move`` should prompt the following traces
     [INFO] [1657870911.985738726] [turtlesim_square_move]: New goal [5.467175 7.581143, 3.123200]
     [INFO] [1657870914.085652821] [turtlesim_square_move]: Reached goal
 
-.. todo: add next steps section when the next tutorial is available.
+Next steps
+----------
+
+Feel free to read the following tutorials extending this one to similar scenarios:
+
+* :ref:`Edge-Cloud TLS communication on WAN <tutorials_cloud_wan_edge_cloud_tls_wan_edge_cloud_tls>`: secure Edge-Cloud communication link by using TLS protocol.
```

Comment:
NIT:
I would point out or make it clear that is not Security in DDS layer, but in TCP layer...